source/Pages/compute_on_spider.rst

```diff
@@ -305,7 +318,408 @@ combination that is not available you will receive the following error message:
 
    'sbatch: error: Batch job submission failed: Requested node configuration is not available'
 
+===============
+Using GPU nodes
```

Comment:
I would propose to split the entire GPU section and make it standalone in the main tree.

Reply:
I agree, it is getting quite big and having a stand alone section allows for adding more details. I decided in some cases to not include certain details because of the size, and I do not want to overwhelm the reader.